internal/handler/router.go

```diff
@@ -0,0 +1,18 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+)
+
+func InitRouter(uh *UserHandler) http.Handler {
+	r := chi.NewRouter()
+	r.Use(middleware.Logger)
+	r.Post("/register", Handle(uh.UserService.RegisterUser))
+	r.Post("/login", Handle(uh.UserService.Login))
+	r.Get("/getUserById", Handle(uh.UserService.GetUserById))
```

Comment:
please read about how to design rest application

https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design